app/models/user.js

```diff
@@ -0,0 +1,48 @@
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define(
+    'user',
+    {
+      id: {
+        type: DataTypes.INTEGER,
+        primaryKey: true,
+        autoIncrement: true
+      },
+      first_name: {
```

Comment:
model properties should be in `camelCase` that's why previously I commented that you should find a way to map the `camelCase` properties of the model to the `snake_case` actual fields of the table